### DISCUSSION TITLE
el heddaf tv algerie url from youtube

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#139] [Show and tell]
https://www.youtube.com/watch?v=aC6XZh6meQg&ab_channel=ElheddafTVCompte

**Comment by BellezaEmporium** [comment]
Should be http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UC8rOLwbg9yW9-Y6NAQSojJQ/live

but they do not stream 24/7.
